generators/client-2/templates/src/main/webapp/app/_app.state.ts

```diff
@@ -1,5 +1,8 @@
 import { NavbarComponent } from './layouts';
-import { AuthService, <%=jhiPrefixCapitalized%>LanguageService } from './shared';
+import { AuthService } from './shared';
+<%_ if (enableTranslation) { _%>
```

Comment:
@deepu105 can we have this only for `html` ? 

`state.js` and `import` can we follow what we had so far.